apps/website/src/data/found-animal.ts

```diff
@@ -242,6 +242,150 @@ const data: FoundAnimalFlow = {
         ],
       },
     },
+    {
+      name: "Squirrel",
+      flow: {
+        prompt: [
+          "Does any of the following apply to the squirrel?",
+          "- It is bleeding, has an open wound, or has a broken bone",
+          "- It has been in a cat's or dog's mouth",
+          "- It is covered in fly eggs (looks like small grains of rice)",
+          "- It is cold, wet, or crying nonstop",
+        ],
+        options: [
+          {
+            name: "Yes",
+            flow: {
+              prompt: macros.general.rehab("the squirrel"),
+            },
+          },
+          {
+            name: "No",
+            flow: {
+              prompt: [
+                "If the squirrel is uninjured the next step is to identify its age to determine if intervention is needed",
+                "Does any of the following apply to the squirrel",
+                "- have a fluffed out tail",
+                "- a body longer than 6 inches (not including the tail)",
+                "- approach humans or pets",
+              ],
+              options: [
+                {
+                  name: "Yes",
+                  flow: {
+                    prompt: [
+                      "This is a juvenile squirrel, you do not need to intervene.",
+                      ...macros.general.leave("the squirrel"),
+                    ],
+                  },
+                },
+                {
+                  name: "No",
+                  flow: {
+                    prompt: [
+                      "This is an infant squirrel, You will need to guide the baby back to its mother",
+                      "Place uncooked rice or bird seed in a sock and warm in the microwave for 20-30 seconds, Wrap the sock in a soft towel, and place it with the baby squirrel in an open container (e.g. a box)",
+                      "Do NOT give the baby food or water",
+                      "Return the squirrel to its nesting tree (usually in the immediate area where the squirrel was found) If you dont know which tree the squirrels nest is in, or if the nest  was destroyed. Then choose a tree closest to where the squirrel was found",
+                      "Observe the baby for the next 6 to 8 hours, Reheat the rice every two hours. If the mother does return after 24 hours, you can...",
+                      ...macros.general.rehab("the squirrel"),
```

Comment:
Instead of having these two directly here, we might want a "Done" confirmation and steps similar to what is done in the bird flow?